cerberus-audit-logger-athena/src/test/java/com/nike/cerberus/audit/logger/service/S3LogUploaderServiceTest.java

```diff
@@ -51,4 +53,35 @@ public void test_that_getPartition_works() {
     String expected = "partitioned/year=2018/month=01/day=29/hour=12";
     assertEquals(expected, actual);
   }
+
+  @Test
+  public void test_that_getPartition_fails() {
+    String fileName = "dummy";
+    String actual = s3LogUploader.getPartition(fileName);
+    String expected = "un-partitioned";
+    assertEquals(expected, actual);
+  }
+
+  @Test
+  public void test_ingest_log_works() {
+    try {
+      String fileName = "localhost-audit.2018-01-29_12-58.log.gz";
+      // File f = new File(fileName);
+      s3LogUploader.ingestLog(fileName);
+      Thread.currentThread().sleep(TimeUnit.SECONDS.toMillis(60));
+      assertTrue(true);
+    } catch (Exception e) {
+      assertTrue(false);
+    }
+  }
+
+  @Test
+  public void test_executor_shutdown_works() {
+    try {
+      s3LogUploader.executeServerShutdownHook();
```

Comment:
This test doesn't actually test that anything happens in that method. You're just asserting that it doesn't throw an exception. What about the actual logic in that method? Are we shutting down the executor correctly? Are we waiting for it to terminate cleanly? Etc.

Reply:
Instead of asserting false on assertTrue to test for an exception or not, you should have the test just catch the exception.

Reply:
made changes

Reply:
What changes were made, this test is still saying `assertTrue(false)`?

Reply:
I think i have changes in my local, i will take a look and verify that